README.md

```diff
@@ -169,82 +191,90 @@ In the following section we guide you trough installing and running a Gazebo sim
    ```
 1. Finally, set the GAZEBO_MODEL_PATH in your bashrc:
-```bash
-echo export GAZEBO_MODEL_PATH=${GAZEBO_MODEL_PATH}:~/catkin_ws/src/avoidance/avoidance/sim/models:~/catkin_ws/src/avoidance/avoidance/sim/worlds >> ~/.bashrc
-```
+   ```bash
+   echo "export GAZEBO_MODEL_PATH=${GAZEBO_MODEL_PATH}:~/catkin_ws/src/avoidance/avoidance/sim/models:~/catkin_ws/src/avoidance/avoidance/sim/worlds" >> ~/.bashrc
+   ```
 
-Steps 3, 4 and 5 together with sourcing your catkin setup.bash (`source ~/catkin_ws/devel/setup.bash`) should be repeated each time a new terminal window is open.
+The last three steps, together with sourcing your catkin **setup.bash** (`source ~/catkin_ws/devel/setup.bash`) should be repeated each time a new terminal window is open.
 You should now be ready to run the simulation using local or global planner.
 
 ### Local Planner (default, heavily flight tested)
 
 This section shows how to start the *local_planner* and use it for avoidance in mission or offboard mode.
 
-The planner is based on the [3DVFH+](http://ceur-ws.org/Vol-1319/morse14_paper_08.pdf) algorithm. To run the algorithm it is possible to
-
-* simulate a forward looking stereo camera running OpenCV's block matching algorithm (SGBM by default)
-
-   ```bash
-   # if stereo-image-proc not yet installed
-   sudo apt install ros-kinetic-stereo-image-proc
-
-   roslaunch local_planner local_planner_stereo.launch
-   ```
-
-   The disparity map from `stereo-image-proc` is published as a [stereo_msgs/DisparityImage](http://docs.ros.org/api/stereo_msgs/html/msg/DisparityImage.html) message, which is not supported by rviz or rqt. To visualize the    message, either run:
+The planner is based on the [3DVFH+](http://ceur-ws.org/Vol-1319/morse14_paper_08.pdf) algorithm.
 
-   ```bash
-   # if image_view is not yet installed
-   sudo apt install ros-kinetic-image-view
-
-   rosrun image_view stereo_view stereo:=/stereo image:=image_rect_color
-   ```
+You *may* need to install some additional dependencies (if not already installed):
+* Melodic:
+  ```sh
+  sudo apt install ros-melodic-stereo-image-proc ros-melodic-image-view
+  ```
+* Kinetic:
+  ```sh
+  sudo apt install ros-kinetic-stereo-image-proc ros-kinetic-image-view
+  ```
+   
+To run the algorithm it is possible to:
 
-   or publish the DisparityImage as a simple sensor_msgs/Image
+* simulate a forward looking stereo camera running OpenCV's block matching algorithm (SGBM by default)
+  ```bash
+  roslaunch local_planner local_planner_stereo.launch
+  ```
+    
+  The disparity map from `stereo-image-proc` is published as a [stereo_msgs/DisparityImage](http://docs.ros.org/api/stereo_msgs/html/msg/DisparityImage.html) message, which is not supported by rviz or rqt. 
+   To visualize the message, either run:
 
-   ```bash
-   rosrun topic_tools transform /stereo/disparity /stereo/disparity_image sensor_msgs/Image 'm.image'
+  ```bash
+  rosrun image_view stereo_view stereo:=/stereo image:=image_rect_color
```

Comment:
@mrivi For this  call and the next one I get `Unable to register with master node [http://localhost:11311]: master may not be running yet. Will keep trying.` (see below).

It is not clear where this runs. Is it in the same terminal as the above call? What happens if I run one of these vs I don't?

```
hamish@hamish-ThinkPad-T470:~/github/Firmware$ rosrun image_view stereo_view stereo:=/stereo image:=image_rect_color
[ERROR] [1581307267.604134449]: [registerPublisher] Failed to contact master at [localhost:11311].  Retrying...
^Chamish@hamish-ThinkPad-T470:~/github/Firmware$ rosrun topic_tools transform /stereo/disparity /stereo/disparity_image sensor_msgs/Image 'm.image' 
Unable to register with master node [http://localhost:11311]: master may not be running yet. Will keep trying.
```

Reply:
-  `roslaunch local_planner local_planner_stereo.launch` is running
- new terminal, `source devel/setup.bash`, `rosrun image_view stereo_view stereo:=/stereo image:=image_rect_color`

Reply:
Thanks. I think this is clarified. Please check on final review.